[2025-03-25T15:13:11.996+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: passanger_traffic.creare1 manual__2025-03-25T15:12:57.822582+00:00 [queued]>
[2025-03-25T15:13:12.012+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: passanger_traffic.creare1 manual__2025-03-25T15:12:57.822582+00:00 [queued]>
[2025-03-25T15:13:12.013+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-25T15:13:12.038+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): creare1> on 2025-03-25 15:12:57.822582+00:00
[2025-03-25T15:13:12.052+0000] {standard_task_runner.py:57} INFO - Started process 245 to run task
[2025-03-25T15:13:12.058+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'passanger_traffic', 'creare1', 'manual__2025-03-25T15:12:57.822582+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/passenger_traffic_dag.py', '--cfg-path', '/tmp/tmpvdhjqv80']
[2025-03-25T15:13:12.079+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask creare1
[2025-03-25T15:13:12.157+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-25T15:13:12.250+0000] {task_command.py:415} INFO - Running <TaskInstance: passanger_traffic.creare1 manual__2025-03-25T15:12:57.822582+00:00 [running]> on host f489f841bffe
[2025-03-25T15:13:12.421+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='passanger_traffic' AIRFLOW_CTX_TASK_ID='creare1' AIRFLOW_CTX_EXECUTION_DATE='2025-03-25T15:12:57.822582+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-25T15:12:57.822582+00:00'
[2025-03-25T15:13:12.425+0000] {sql.py:274} INFO - Executing: 
    CREATE SCHEMA IF NOT EXISTS presentation;
[2025-03-25T15:13:12.441+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-25T15:13:12.458+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-25T15:13:12.478+0000] {sql.py:418} INFO - Running statement: 
    CREATE SCHEMA IF NOT EXISTS presentation;, parameters: None
[2025-03-25T15:13:12.499+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=passanger_traffic, task_id=creare1, execution_date=20250325T151257, start_date=20250325T151311, end_date=20250325T151312
[2025-03-25T15:13:12.537+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-25T15:13:12.585+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
